support-frontend/assets/pages/[countryGroupId]/thank-you.tsx

```diff
@@ -152,16 +305,16 @@ export function ThankYou({ geoId }: Props) {
 		return <div>Unable to find contribution type {contributionType}</div>;
 	}
 
-	const isOneOff = order.product === 'Contribution';
+	const isOneOff = productKey === 'Contribution';
```

Comment:
Is this correct? I thought `productKey === 'Contribution'` for recurring contributions? Do we need this `isOneOff` logic in the generic TY page at that moment?

Reply:
This is a good point - I'll do that as a secondary fix to make sure it's not conflated.

Reply:
Yeah I was worried it may introduce an unexpected behaviour for users paying via paypal when taking out a recurring contribution, it's just used for the `isOneOffPayPal` prop passed to `ThankYouHeader` and it looks like it just alters the header copy - I don't think this PR is introducing a change to that behaviour though. So yes probably something we can look at as a secondary fix.

Reply:
Yea, so I think we should merge this in and then check current functionality and see we haven't coupled this to anything where what we actually mean is `isContribution`.